src/metabase/driver/postgres.clj

```diff
@@ -56,19 +57,15 @@
 
 (defmethod driver/display-name :postgres [_] "PostgreSQL")
 
-(defmethod driver/database-supports? [:postgres :persist-models]
-  [_driver _feat _db]
-  true)
-
-(defmethod driver/database-supports? [:postgres :persist-models-enabled]
-  [_driver _feat db]
-  (-> db :options :persist-models-enabled))
-
-(doseq [feature [:actions :actions/custom]]
+(doseq [[feature supported?] {:persist-models         (constantly true)
+                              :convert-timezone       (fn [driver _db] (= driver :postgres))
+                              :persist-models-enabled (fn [_driver db] (-> db :options :persist-models-enabled))
+                              ;; actions only supported for Postgres for right now. Not supported for child drivers like Redshift or whatever.
+                              :actions                (fn [driver _db] (= driver :postgres))
+                              :actions/custom         (fn [driver _db] (= driver :postgres))}]
   (defmethod driver/database-supports? [:postgres feature]
-    [driver _feat _db]
-    ;; only supported for Postgres for right now. Not supported for child drivers like Redshift or whatever.
-    (= driver :postgres)))
+    [driver _feature database]
+    (supported? driver database)))
```

Comment:
I'm having trouble convincing myself this is an improvement over the way it was before.

Reply:
Will change it back.

Reply:
Will change it back.